uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -577,17 +577,17 @@ Feature: GGMQ-1
       | mqtt-v | name     | agent                                    | recipe               | retainHandling-2 |
       | v3     | sdk-java | aws.greengrass.client.Mqtt5JavaSdkClient | client_java_sdk.yaml | true             |
 
-    @mqtt3 @mosquitto-c
```

Comment:
why is mosquitto removed? Shouldn't this test run with both paho and mosquitto?

Reply:
test doesn't support mosquitto-c yet